src/libs/fsdrv/lv_fsdrv.h

```diff
@@ -48,6 +48,8 @@ void lv_fs_memfs_init(void);
 #endif
 
 #if LV_USE_FS_LITTLEFS
+#include "lfs.h"
+void lv_littlefs_set_handler(lfs_t *);
```

Comment:
Do `"lfs.h"` and `lv_littlefs_set_handler `need to be exposed to public space?

Reply:
Yes, otherwise it's not possible to call them as it's declared in the c file.

Reply:
How about this?

```c
struct lfs;
void lv_littlefs_set_handler(struct lfs *);
```

This does not expose the littlefs API (`"lfs.h"`) to public space and is more concise.

Reply:
Oh, wasn't aware of this possibility, thank you again @FASTSHIFT :pray: 

Really sorry for being such a beginner :/